tests/src/test/java/alluxio/shell/command/CopyFromLocalCommandTest.java

```diff
@@ -406,4 +406,14 @@ public void copyFromLocalWildcard() throws IOException {
     Assert.assertTrue(fileExists(new AlluxioURI("/testDir/foobar3")));
     Assert.assertFalse(fileExists(new AlluxioURI("/testDir/foobar4")));
   }
+
+  @Test
+  public void copyFromLocalRelativePath() throws IOException {
+    File localDir = new File("localDir");
```

Comment:
@aaudiber  I have noticed that since I wrote the test, thanks for your information.